static/js/views/settings.js

```diff
@@ -1520,13 +1520,14 @@ const SettingsScreen = {
       const addonList = document.getElementById('installed-addons-list');
       addonList.innerHTML = '';
 
-      for (const name of Array.from(this.installedAddons.keys()).sort()) {
-        const addon = this.installedAddons.get(name);
-        components.set(
-          name,
-          new InstalledAddon(addon, this.installedAddons, this.availableAddons)
-        );
-      }
+      Array.from(this.installedAddons.entries())
+        .sort((a, b) => a[1].display_name.localeCompare(b[1].display_name))
+        .forEach((x) => {
+          components.set(
+            x[0],
```

Comment:
nit: x could be named addon to make this a bit clearer

Reply:
`x` is actually a 2 element array, which are `name` and `addon`.